grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/BulkResourceMutate.java

```diff
@@ -107,6 +107,12 @@ void flush() {
                 hasFailed = true;
                 numberOfFailures++;
                 LOGGER.info("Number of failures: " + numberOfFailures);
+                LOGGER.info("Backing Off");
+                try {
+                    Thread.sleep(1000);
+                } catch (InterruptedException e1) {
+                    e1.printStackTrace();
+                }
```

Comment:
Yes it can go in for now, I have a TP task to make it an exponential backoff on the retry